### PR TITLE
🐳 Add local-db

### LINKED ISSUE #415: implement easier local MongoDB dev environment setup
While getting the `upswyng-web` project up-and-running is fairly simple, it's not so easy to start the provider portal (`upswyng-server` project).

One hurdle to overcome is providing an easy way to setup a local Mongo DB database. While we do have a dev database keys we can share with developers upon request, it would be best to have an easy way to setup this database locally.

- How would we go about seeding that database?
- What resources or resource placeholder would exist?
- Should they be seeded with users or various roles? (admin vs. logged in user)

### PR DESCRIPTION
**Pull request by jollyjerr**
This PR closes #415 

## What does this PR do?

Adds a development database docker image that forwards the connection to a local port. This database is seeded using [mongorestore](https://docs.mongodb.com/manual/reference/program/mongorestore/) from a mongodump of the dev database.

I also added documentation and startup scripts, so a dev simply needs to run `yarn build:local-db` to get started.

The dev-db comes populated with a super-admin gmail user 
```
username: dev.upswyng@gmail.com
password: upswyng123
````

If we don't like the preseeded admin user approach, I could also write documentation for how to give your own user admin privileges with the mongo-express dashboard included in this PR

### CODE REVIEW

**Comment by jollyjerr** [comment]
I am going to make some changes to this pending #429

**Comment by jollyjerr** [comment]
Hey @jacobvenable or @rhinodavid this is ready for review. Documentation can be found in the server README, but if you want to build the db locally just run `yarn build:local-db` while docker is running on your machine. Server's `.env.example` has been updated with connection information.

File: packages/server/README.md [hunk 1]
```diff
@@ -9,11 +9,23 @@ This workspace includes:
...
+
+From your root directory...
+
+```
+yarn build:local-db
+```
+
+You can start and stop the local database via your docker dashboard
+
+![](./readme_docker.png)
```
Comment (jacobvenable) [inline packages/server/README.md:26]:
```suggestion
![example dashboard](./readme_docker.png)
```

File: packages/server/README.md [hunk 2]
```diff
@@ -44,6 +56,15 @@ yarn dev
 
 Runs the app in the development mode. Open [http:/localhost:3000](http:/localhost:3000) to view it in the browser.
 
+If you want to work on the [provider portal](http://localhost:3000/provider), you can log in with this google user:
+
+```
+username: dev.upswyng@gmail.com
```
Comment (jacobvenable) [inline packages/server/README.md:62]: We should make a follow-up to this to make it a @codeforboulder.org email for easier management. Not needed for this PR though!

**Review by jacobvenable — APPROVED** [review]
It's running smooth :D This is a big improvement to our current setup! Noice job!

File: packages/server/README.md (hunk 2 above)
Comment (jollyjerr) [reply packages/server/README.md:62]: 👍

**Pull request merged**